app/assets/javascripts/components/cloud_volume_backup/cloud-volume-backup-form-component.js

```diff
@@ -1,4 +1,15 @@
-ManageIQ.angular.app.controller('cloudVolumeBackupFormController', ['miqService', 'cloudVolumeBackupFormId', '$http', function(miqService, cloudVolumeBackupFormId, $http) {
+ManageIQ.angular.app.component('cloudVolumeBackupFormComponent', {
```

Comment:
`cloudVolumeBackupForm` please .. no need to call every component FooComponent :)